docs/glossary.md

```diff
@@ -0,0 +1,42 @@
+| Amenities | Additional services or features offered to enhance the experience of the passengers, such as food, entertainment, Wi-Fi, extra legroom, baggage allowance, frequent flyer programs, and lounge access. They can vary depending on the class of service and the airline/train company. |
+| Baggage allowance | The amount of luggage that a passenger is allowed to carry on a flight without additional charges. |
+| Booking | The process of reserving a seat on a flight or a room in a hotel. |
+| Cabin | The section of an aircraft or train where passengers sit during their trip. It is divided into different classes, such as first class, business class, and economy class, each one with different amenities and prices. |
+| Commission | Fee paid to intermediaries for booking travel-related services, usually a percentage of the total cost. |
+| Carrier Code          | 2 to 3-character IATA carrier code (IATA table codes).  |
+| Country Code          | Country code following ISO 3166 Alpha-2 standard.         |
+| Direct flight | A flight that goes from one destination to another without any stops in between. |
+| Fare | The price of a ticket for a particular flight or travel itinerary. |
+| Fare Rules | The terms and conditions that apply to a specific airline ticket or fare, including restrictions and information on refunds, cancellations, changes, baggage, seat assignments, upgrades, and frequent flyer programs. |
+| Flight Order Id       | Unique identifier returned by the [Flight Create Orders API](https://developers.amadeus.com/self-service/category/air/api-doc/flight-create-orders). |
+| GDS (Global Distribution System) | A computerized system used by travel agents and airlines to search for and book flights, hotels, rental cars, and other travel-related services |
+| GeoSure GeoSafeScores | Algorithm that analyzes crime, health and economic data, official travel alerts, local reporting and a variety of other sources.    |
+| Hotel Ids             | Amadeus Property Codes (8 chars). Comma-separated list of Amadeus Hotel Ids (max. 3). Amadeus Hotel Ids are found in the Hotel Search response (parameter name is `hotelId`).                                                     |
+| IATA | [International Air Transport Association](www.iata.org) |
+| IATA Code             | [Code](https://www.iata.org/en/publications/directories/code-search/) used by IATA to identify locations, airlines and aircraft.     |
+| ICAO |  [International Civil Aviation Organization](https://www.icao.int/) |
+| ISO8601 date format               | PnYnMnDTnHnMnS format, e.g. PT2H10M.      |
+| Layover | A stopover in a destination en route to the final destination. |
+| Location Id           | Amadeus-defined identifier that you can see in the search results when querying the Safe Place API by latitude/longitude/radius or square information.                                                                            |
```

Comment:
Location Id can also be found in other APIs, such as [Airport and City Search](https://developers.amadeus.com/self-service/category/air/api-doc/airport-and-city-search/api-reference)